Makefile

```diff
@@ -5,9 +5,19 @@ watch: ## Start a file watcher to run tests on change. (requires: watchexec)
 .PHONY: all
 all: lint test build ## test -> lint -> build
 
+.PHONY: deps
+deps: ## Setup or Update linters
+	go get -u github.com/kisielk/errcheck
+	go get -u golang.org/x/lint/golint
+	go get -u github.com/securego/gosec/cmd/gosec
+	go get -u honnef.co/go/tools/cmd/staticcheck
+	go mod tidy
```

Comment:
The `-u` will install the binaries on users' `$GOPATH/bin` and because of the `go get` the modules are added as project dependencies and hence the last command `go mod tidy` to remove them.
I think it's the simplest approach to download them, any other solution will require users' OS dependency.
I don't see any problem with these.

Reply:
As per the documentation:
> The -u flag instructs get to use the network to update the named packages
and their dependencies. By default, get uses the network to check out
missing packages but does not use it to look for updates to existing packages.

This has nothing to do with the gopath.
What about a `cd ; go get everything ; cd -`
If you are worried about an OS not supporting `cd`, do these OSes support `make`? Do we support them? because we only support win32/64 linux32/64 darwin64

Reply:
Also, is `gotestsum` downloaded here?

Reply:
> This has nothing to do with the gopath.
What about a cd ; go get everything ; cd -

You are right. But the `-u` does also install the module's binary into the `GOBIN`, I was merely pointing the bin directory.
And yes, cd & cd revert will be even better.

Reply:
Then let's solve it with `cd` and no `go mod tidy`